modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -265,51 +259,55 @@ The following properties are available and can be used in the alert Email to, Su
 
 == Editing Existing Alerts
 
-If you view the panel for an application or server, you will see a *View Alerts* link. In there you will be able to see and manage the alerts that are sourced from it. You can also click on the *Alerts* link on the left panel to view all alerts together.
+If you view the panel for an application or server, you see a *View Alerts* link, where you can see and manage the alerts that are sourced from it. You can also click the *Alerts* link in the left panel to view a list of all alerts.
 
 image::view-alerts.png[alerts]
 
 
-From this menu you can filter the displayed alerts, click on an alert name to directly edit it, or click on the icon next to one to open its panel to view its settings. From this panel you can click the *edit* icon next to the name to edit the same options you have when creating a new one.
+From this menu you can filter the displayed alerts, click on an alert name to directly edit it, or click the icon next to one to open its panel to view its settings. From this panel you can click the *edit* icon next to the name to edit the same options you have when creating a new alert.
 
 image::edit-alert.png[edit]
 
 
 
 === Switching an Alert Off
 
-Select one or multiple alerts in the alert menu and click the switch to change its state form *on* to *off* or viceversa.
+Select one or multiple alerts in the alert menu and click the switch to change its state form *on* to *off* or vice versa.
 
 image::alert-on.png[on]
 
-All users of the Anypoint Platform, even those without permissions to create alerts, can switch the existing alerts that are already created into an active or inactive state. This determines what email alerts will reach their inbox.
+All users of Anypoint Platform, even those without permissions to create alerts, can switch existing alerts into an active or inactive state. This determines what email alerts will reach their inbox.
 
 [NOTE]
 When an alert is disabled or enabled the change is reflected for all users.
```

Comment:
this seems to contradict what is said above, that the user is controlling just his or her own inbox? If the note is true, we should rewrite the description in 279 to make that clear.